### PR TITLE
[WIP] First structure version for cmd sztp agent

### PR DESCRIPTION
**Pull request by alknopfler**
Enable first structure for cmd sztp agent 
Signed-off-by: Alberto Morgante Medina <alknopfler@users.noreply.github.com>

### CODE REVIEW

**Review by mestery — COMMENTED** [review]
Thanks for this PR! Can you add copyright headers to the files before we merge this? It seems like a straightforward change which is using an existing Golang library to execute commands to a sZTP agent on a host otherwise, which is nice!

File: examples/sztp/sztp-agent/cmd/daemon.go [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package cmd
+
+import (
+	"github.com/alknopfler/opi-prov-life/examples/sztp/sztp-agent/pkg/secureAgent"
```
Comment (glimchb) [inline examples/sztp/sztp-agent/cmd/daemon.go:4]: github org needs to change, obviously

**Comment by glimchb** [comment]
@gupta-alok @achilikin fyi

**Comment by alknopfler** [comment]
if a new repo will be created...maybe we could close this PR @glimchb WDYT?

**Comment by alknopfler** [comment]
Close. It's moved to https://github.com/opiproject/sztp/pull/15

**Pull request closed without merging**